sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -209,20 +209,22 @@ class ExpressionParserSuite extends AnalysisTest {
 
   test("binary arithmetic expressions") {
     // Simple operations
-    assertEqual("a * b", 'a * 'b)
-    assertEqual("a / b", 'a / 'b)
+    assertEqual("a * b", UnresolvedMultiply('a, 'b))
```

Comment:
we can update the `assertEqual` method to resolve `UnresolvedAdd` etc. manually.